test/unittests/NDB_BVL_Instrument_Test.php

```diff
@@ -720,7 +722,7 @@ function testScoreElement()
         );
         $this->_instrument->addScoreColumn(
             "FieldName2",
-            null
+            "Field 2",
```

Comment:
After adding the type to $this->_instrument phan picked up on the argument type mismatch. (neither argument to addScoreColumn in NDB_Page is nullable according to the phpdocs.)